tasks/manager/src/test/java/org/infinispan/tasks/SecurityTaskManagerTest.java

```diff
@@ -91,7 +91,6 @@ public Void run() {
 
     @Test(dataProvider = "principleProvider")
```

Comment:
Hey, ```principleProvider``` seems misspelled :)

Reply:
Not relevant to this PR. Create a jira if needed ;)

Reply:
No, I'll just fix it myself and push it when nobody's looking ;)